### PR TITLE
adding game warning

### PR DESCRIPTION
**Pull request by crutkas**

### CODE REVIEW

**Comment by PRMerger7** [comment]
@crutkas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Court72** [comment]
@mattwojo 

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"

**Comment by mattwojo** [comment]
#sign-off

File: hub/powertoys/keyboard-manager.md [hunk 1]
```diff
@@ -167,7 +167,7 @@ Currently Mac and Linux profiles are not included.
 
 ### Will this work on video games?
 
-It depends on how the game accesses your keys. Certain keyboard APIs do not work with Keyboard Manager.
+We do not suggest using Keyboard Manager when playing games as it may affect the game's performance. It will also depend on how the game accesses your keys. Certain keyboard APIs do not work with Keyboard Manager.
```
Comment (Jay-o-Way) [inline hub/powertoys/keyboard-manager.md]: Nit/semantics; difference between "We do not suggest using..." and "We suggest not to use..."
(I think the latter is better)

**Comment by PRMerger17** [comment]
@mattwojo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: hub/powertoys/keyboard-manager.md (hunk 1 above)
Comment (mattwojo) [reply hub/powertoys/keyboard-manager.md]: Reworded. Thanks!

**Pull request merged**